armi/tests/Godiva-blueprints.yaml

```diff
@@ -3,26 +3,6 @@ nuclide flags:
     burn: false
     xs: true
     expandTo: []
-  CL35:
-    burn: false
-    xs: true
-    expandTo: []
-  CL37:
-    burn: false
-    xs: true
-    expandTo: []
-  AL27:
-    burn: false
-    xs: true
-    expandTo: []
-  B10:
-    burn: false
-    xs: true
-    expandTo: []
```

Comment:
Sorry, what is the benefit of removing all these things?

Reply:
I overdefined the number of nuclides flags in the input. I have a personal set I use for my `day job`, that includes many more nuclides than needed for the Godiva unit test. This is essentially trimming a lot of fat.